pkg/codegen/testing/test/testdata/output-funcs-tfbridge20/nodejs/getAmiIds.ts

```diff
@@ -72,10 +71,14 @@ export interface GetAmiIdsResult {
     readonly sortAscending?: boolean;
 }
 
+/**
+ * Taken from pulumi-AWS to regress an issue
+ */
+/** @deprecated aws.getAmiIds has been deprecated in favor of aws.ec2.getAmiIds */
```

Comment:
Where did we depreciate this method? I don't see the code change that did this.

Reply:
This is because output-versioned invokes didn't have doc comments on them which I fixed in this PR, but it's now part of #11511 so it will go away from here